Dockerfile.cloudwatcher

```diff
@@ -0,0 +1,56 @@
+###############################
+# STAGE 1 test and build binary
+###############################
+
+FROM golang:1.12 AS builder
+
+# Set Go build env vars
+ARG GOOS=linux
```

Comment:
Why declare these on their own line?  I don't think they're being used anywhere below

Reply:
These are part of the [standard `go build` env vars](https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63). I'm being explicit in case this is built on Mac/Windows/whatever (`go build` defaults to whatever OS/Arch it's running on). This will, for example, let @alphachai build and run this on his MacBook.